__tests__/app.test.js

```diff
@@ -80,6 +80,45 @@ describe("GET /api/articles", () => {
         });
       });
   });
+  test("200: returns an object with an array with correctly formatted objects sorted by any column in the table", () => {
```

Comment:
you advent really tested here the order queries are working as intended, although you have handled the error cases